src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.DeadlineCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.EventCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.IncorrectCommand;
+import duke.command.ListCommand;
+import duke.command.TodoCommand;
+
+public class Parser {
+    /**
+     * Parses a string to a Command
+     *
+     * @param fullCommand the command in string
+     * @return            the command in Command class
+     */
+    public static Command parse(String fullCommand) {
+        String[] inputValues = fullCommand.split(" ", 2);
+        String commandInstruction = inputValues[0];
+
+        String parameter1 = getParameters(inputValues)[0];

```

Comment:
Very neat and efficient code here, well done! Might I suggest replacing the varables "parameter1" and "parameter2" with variables like "taskName" and "taskTime" to slightly improve code readability?